components/atoms/tooltip/index.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import Popover from 'react-popover'
+
+import styles from './styles.css'
+
+class Tooltip extends React.Component {
```

Comment:
Import Component

Reply:
It's not wrong but we're importing `Component` so I think it's important to keep the consistency